ansible/roles/stack-sunbird/templates/transformer.env

```diff
@@ -52,4 +52,4 @@ NETCORE_WHATSAPP_SOURCE={{uci_netcore_whatsapp_source}}
 NETCORE_WHATSAPP_URI={{uci_netcore_whatsapp_uri | default('https://waapi.pepipost.com/api/v2/')}}
 
 #Telemetry
-TELEMETRY_EVENT_PDATA_ID={{env}}.uci.{{sunbird_instance}}
\ No newline at end of file
+TELEMETRY_EVENT_PDATA_ID={{uci_env}}.uci.{{sunbird_instance}}
```

Comment:
Please add a defult like this
`TELEMETRY_EVENT_PDATA_ID={{uci_env | default(env)}}.uci.{{sunbird_instance}}`